test/gtest/cba_find2.hpp

```diff
@@ -0,0 +1,187 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2022 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+#pragma once
+
+#include <random>
+
+#include <gtest/gtest.h>
+#include <miopen/miopen.h>
+#include <miopen/problem.hpp>
+#include <miopen/solver_id.hpp>
+#include <miopen/fusion/fusion_invoke_params.hpp>
+#include <serialize.hpp>
+#include <fusionHost.hpp>
+
+#include "tensor_util.hpp"
+#include "get_handle.hpp"
+#include "conv_common.hpp"
+
+#include "conv_test_base.hpp"
+#include "conv_tensor_gen.hpp"
+
+template <typename T = float>
+struct ConvBiasActivInferFind2Test
+    : public ::testing::TestWithParam<
+          std::tuple<miopenActivationMode_t, ConvTestCase, miopenTensorLayout_t>>,
+      ConvFwdSolverTestBase<T>
+{
+protected:
+    ConvTestCase conv_config;
+    miopen::ActivationDescriptor activ_desc;
+    tensor<T> bias;
+    miopen::Allocator::ManageDataPtr bias_dev;
```

Comment:
Would it help if this were a `tensor<T>` instead so that it has a descriptor attached to it.

Reply:
This is 1 to 1 copy from another test. I can change it, but the purpose was to introduce the least amount of changes possible outside of the PR scope. I can change both tests if you want.